pkg/api/errors.go

```diff
@@ -0,0 +1,17 @@
+package api
+
+import (
+	"errors"
+)
+
+var (
+	// ErrEnsureStorageNamespace is an error returned when trying to create a new repository
+	// and the attempt to access the storage fails
+	ErrEnsureStorageNamespace = errors.New("failed to ensure access to the storage")
```

Comment:
I think that we didn't fail to ensure - we failed to find a matched storage namespace.
Error like - mismatch storage namespace or something like it describes the issue.